add-chain/main.go

```diff
@@ -205,14 +214,12 @@ func getGasPayingToken(l1rpcURl string, SystemConfigAddress superchain.Address)
 
 	opts := bind.CallOpts{}
 	result, err := sc.GasPayingToken(&opts)
-
-	if strings.Contains(err.Error(), "execution reverted") {
-		// This happens when the SystemConfig contract
-		// does not yet have the CGT functionality.
-		return nil, nil
-	}
```

Comment:
This caused the program to crash when `err == nil`

Reply:
that makes sense 😅